src/main.ts

```diff
@@ -28,6 +29,10 @@ let perfCounters: PerformanceCounterMetrics;
  * @param options Configuration
  */
 export function useAzureMonitor(options?: AzureMonitorOpenTelemetryOptions) {
+    process.env[AZURE_MONITOR_STATSBEAT_FEATURES] = JSON.stringify({
```

Comment:
I would assume some the statbeat features are actually set after the configuration is done right? are you not missing that here?

Reply:
We don't currently set any other statsbeat features from the shim, but setting this after initializing our config avoids the possibility of a future bug. I'll move this.

Reply:
We discussed this need to be there because Distro need to have this before initialization. Please add comment so we don't forget in the future